### PR TITLE
Refactored device room components to use the flux pattern using the reflux implementation

### PR DESCRIPTION
**Pull request by larsblumberg**

### CODE REVIEW

File: public/app/stores/roomStore.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var React = require('react');
+var Reflux = require('reflux');
+
+var roomActions = require('../actions/roomActions');
+
+// TODO: Is this the correct naming style for global variables?
```
Comment (mr-mig) [inline public/app/stores/roomStore.js]: Well, we do not have the naming convention for global vars as **there must not be any global vars** in app code.

The only case we have where global variable is needed - `config` sharing between backend and frontend

File: public/app/stores/roomStore.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    },
+
+    updateRooms: function(rooms) {
+        _rooms = rooms;
+        // Send the updated rooms to all listening components
+        this.trigger(rooms);
+    }
+});
+
+module.exports = roomStore;
```
Comment (mr-mig) [inline public/app/stores/roomStore.js]: If you are using modules, then your "global variables" are not global - they are scoped by the module. They are "module-local"

File: public/app/components/devices/devicespage/devicesPage.jsx [hunk 3]
```diff
@@ -1,24 +1,41 @@
 'use strict';
 
 var React = require('react');
-
+var Reflux = require('reflux');
 var Room = require('../room/room.jsx');
 
+var roomStore = require('../../../stores/roomStore');
```
Comment (dev-tim) [inline public/app/components/devices/devicespage/devicesPage.jsx]: it's better to use alias here

File: public/app/stores/roomStore.js (hunk 1 above)
Comment (dev-tim) [inline public/app/stores/roomStore.js]: We could move it to stub.json file

**Comment by mr-mig** [comment]
Shall we merge this?

**Pull request closed without merging**